security_drone_user_app/lib/presentation/pages/view_thief_page.dart

```diff
@@ -0,0 +1,140 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:security_drone_user_app/logic/thief_page_bloc.dart';
+import 'package:security_drone_user_app/presentation/text_section.dart';
+
+import '../image_banner.dart';
+
+class ViewThiefPage extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() {
+    return ViewThiefPageState();
+  }
+}
+
+class ViewThiefPageState extends State<ViewThiefPage>{
+
+  @override
+  Widget build(BuildContext context){
+    // ignore: close_sinks
+    ThiefPageBloc _bloc = ThiefPageBloc();
+
+    return Scaffold(
+      body: ThiefListClass(_bloc),
+      floatingActionButton: Column(
+        mainAxisSize: MainAxisSize.min,
+        children: <Widget>[
+          FloatingActionButton(
+            child: Icon(Icons.refresh),
+            onPressed: () => _bloc.add(
+              RefreshThiefEntries(),
+            ),
+          ),
+        ],
+      )
+    );
+  }
+}
+
+// ignore: must_be_immutable
+class ThiefListClass extends StatelessWidget {
+  ThiefPageBloc _bloc;
+
+  ThiefListClass(this._bloc);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
```

Comment:
The sacffold with the appbar should be in the main of the application. This should not be implemented in every widget.